relay-metrics/src/aggregation.rs

```diff
@@ -1388,15 +1437,16 @@ impl<T: Iterator<Item = Bucket>> FusedIterator for CappedBucketIter<T> {}
 /// all elapsed buckets belonging to the same [`ProjectKey`] are flushed together.
 ///
 /// Receivers must implement a handler for the [`FlushBuckets`] message.
-pub struct Aggregator {
+pub struct AggregatorService {
     config: AggregatorConfig,
     buckets: HashMap<BucketKey, QueuedBucket>,
     receiver: Recipient<FlushBuckets>,
```

Comment:
`Recipient` is still something actix, why is that still here?

Reply:
This service still interacts with `project_cache` actor which runs on the old actix.  I wanted to keep these changes to the minimum and then when the `project_cache` is converted we can remove this `Recipient` and maybe replace it with `mpsc` channel? 

I can look into it and maybe introduce changes in this PR, if that's better for the progress of migration.

Reply:
the approach has been to use compat senders during the transition, I think that makes the most sense here as well.  the code here will become easier to grasp if it is fully converted